app/src/full/java/io/homeassistant/companion/android/sensors/AndroidAutoSensorManager.kt

```diff
@@ -90,11 +90,17 @@ class AndroidAutoSensorManager : SensorManager, Observer<Int> {
                 false to "Unknown($value)"
             }
         }
+
+        val icon = when (connected) {
+            true -> "mdi:car"
+            else -> "mdi:car-off"
+        }
+
         onSensorUpdated(
             context,
             androidAutoConnected,
             connected,
-            androidAutoConnected.statelessIcon,
+            icon,
```

Comment:
Use a pattern that is also found in other boolean sensors:

```suggestion
            if (connected) androidAutoConnected.statelessIcon else "mdi:car-off"
```